src/include/ipc/info.h

```diff
@@ -54,9 +54,10 @@ struct sof_ipc_fw_version {
 	uint8_t time[10];
 	uint8_t tag[6];
 	uint32_t abi_version;
+	uint32_t fw_hash; /* used to check FW and ldc file compatibility */
```

Comment:
how is this different from git version hash ?

Reply:
`SOF_TAG` refer git commit hash, changed after each commit, so I can edit file without commit, what wouldn't change `SOF_TAG` but will break ldc file compatibility. `fw_hash` will change value after each edit of any file tracked in git (response for following question).

Reply:
ok, so this need to be made clear in the comments alongside it's reproducability.

Reply:
added